src/features/courses/course-controller.ts

```diff
@@ -202,6 +208,21 @@ class CourseController {
                 });
             }
 
+            if (includeGradeIdsThatNeedRegrade) {
+                questionSubInclude.push({
+                    model: StudentGrade,
+                    as: 'grades',
+                    required: false,
+                    attributes: ['id'],
+                    where: {
+                        [sequelize.Op.and]: [
```

Comment:
Is this `and` necessary? It doesn't seem to be doing anything.

Reply:
It is necessary because there are an array of conditions for the same property

Also this is wrong and should be an or